core-data/entities.js

```diff
@@ -5,7 +5,8 @@ import { find, upperFirst, camelCase } from 'lodash';
 
 const entities = [
 	{ name: 'postType', kind: 'root', key: 'slug', baseUrl: '/wp/v2/types' },
-	{ name: 'media', kind: 'root', baseUrl: '/wp/v2/media' },
+	{ name: 'media', plural: 'mediaItems', kind: 'root', baseUrl: '/wp/v2/media' },
+	{ name: 'taxonomy', kind: 'root', key: 'slug', baseUrl: '/wp/v2/taxonomies', plural: 'taxonomies' },
```

Comment:
Nit: Inconsistent property ordering.

- Previous line: `name`, `plural`, `kind`, `baseUrl`
- This line: `name`, `kind`, `key`, `baseUrl`, `plural`